src/parser/core/Combatant.ts

```diff
@@ -105,13 +96,16 @@ class Combatant extends Entity {
     };
 
     this._parseTalents(combatantInfo.talents);
-    this._parseTraits(combatantInfo.artifact);
-    this._parseEssences(combatantInfo.heartOfAzeroth);
-    this._parseCovenant(combatantInfo.covenant);
-    this._parseSoulbind(combatantInfo.soulbind);
-    this._parseConduits(combatantInfo.conduits);
+    this._parseTraits(combatantInfo.artifact as Trait[]);
+    this._parseEssences(combatantInfo.heartOfAzeroth as Essence[]);
```

Comment:
should these be in an `else` after checking for shadowlands expansion? I havent looked at the actual combatantinfo shape, but I would assume if the artifact/hoa are essences *or* conduits it might be weird to parseEssences using the conduit data?

Reply:
I planned on just deleting them when we swap over to non-prepatch since they're then irrelevant